ex6B/ex6Spec.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Write a string splitter: given a string of comma separated values, would
+ * return an array with the values.
+ *
+ * For example:
+ * split('a,b,c') === ['a', 'b', 'c']
+ * split('hello,world') === ['hello', 'world']
+ * split('hello') === ['hello']
+ * split('hello,,,world') === ['hello', '', '', 'world']
+ * split('') === []
+ */
+var ex6 = require('./ex6.js');
+
+describe("split", function() {
+
+	it("returns 3 chracters", function() {
+		expect(ex6.split('a,b,c')).toBe('a,b,c');
```

Comment:
As said in the previous pull request, this is incorrect. Instead, your method should return an array and you should check against that.